docs-source/documentation/advanced-scenarios/browsers.md

```diff
@@ -0,0 +1,50 @@
+# Browsers
+
+The SDK is configured to use a platform specific browser implementation to handle the redirect to Auth0. Depending on the platform, one or more Browser implementations might be available.
+An example is the Auth0.OidcClient.iOS SDK, which auto selects the browser based on the version of iOS used.
+
+Even though we do not recommend changing this, you could ensure all versions of iOS use the same browser by passing an explicit browser instance to the Browser property of the `Auth0ClientOptions`:
+
+```
+var client = new Auth0Client(new Auth0ClientOptions {
+    Domain = "YOUR_AUTH0_DOMAIN",
+    ClientId = "YOUR_AUTH0_CLIENT_ID",
+    Browser = new SFSafariViewControllerBrowser()
+});
+```
+
+In case the default implementations that each SDK provides does not suit the needs for your application, each SDK allows to provide your own custom browser implementation by implementing the `IBrowser` interface.
```

Comment:
```suggestion
In case the default implementations that each SDK provides does not suit the needs for your application, each SDK allows you to specify your own custom browser implementation by implementing the `IBrowser` interface.
```